ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+
+## Preamble
+
+```
+SEP: 00XX
+Title: Fiat-To-Fiat payments
+Author: SDF
+Status: Draft
+Created: 2020-04-07
+Updated: 2020-04-07
+Version 1.0.0
+```
+
+## Simple Summary
+
+This SEP defines a protocol for enabling fiat to fiat payments between two real world accounts, facilitated by two different anchors.
```

Comment:
Are there other "real world accounts" other than bank accounts? If not I'd replace the "real world" description to "bank".

Reply:
cash out at a place like moneygram for example

Reply:
Ok good point, how about "non-stellar account"? What I'm trying to get at is we should probably not reinforce the idea that cryptocurrency isn't "real money" by including it in our SEPs.

Reply:
How about:
```suggestion
This SEP defines a protocol for enabling fiat to fiat payments between two financial accounts that exist outside of the Stellar network. The payments are facilitated by two anchors.
```